### DISCUSSION TITLE
Page Number

### DISCUSSION THREAD

**Discussion by pqhf5kd** [bookwyrm-social/bookwyrm#3159] [General]
How should we enter page numbers? The total number of physical pages? The highest numbered page? What about lettered pages?

**Comment by mouse-reeve** [comment]
I think highest numbered page? There's no scenario where people should have to count physical pages :)

**Comment by pqhf5kd** [comment]
https://openlibrary.org/help/faq/editing
OpenLibrary has a short explanation, although I don't find their guidance on unnumbered pages clear.

I have come across records that, for example, state 450 pages and upon checking my copy, I find the highest numbered page 499 and one unnumbered page. 

I think it could be useful to specify highest numbered page.

But what about lettered/roman numerical pages?

I've also seen records that combine the number of lettered and numbered pages
